src/ui/components/shared/SettingsModal/SettingsBodyItem.tsx

```diff
@@ -27,7 +28,7 @@ function Checkbox<K>({ item, value, onChange }: InputProps<K, boolean>) {
     return <span className="italic text-gray-500">Coming Soon...</span>;
   }
 
-  return <input type="checkbox" id={String(key)} checked={value} onChange={toggleSetting} />;
+  return <_Checkbox id={String(key)} checked={value} onChange={toggleSetting} />;
```

Comment:
Naming is tough here 😄 . What do you think about changing the local `CheckBox` to `CheckBoxSetting`?

Reply:
Yep let's do that!